src/views/ReferencesView.tsx

```diff
@@ -0,0 +1,76 @@
+import { ReferenceItem } from '../types/ReferenceItem';
+
+export function ReferencesView({
+    onRefClicked,
+}: {
+    onRefClicked?: (item: ReferenceItem) => any;
+}) {
+    return (
+        <div>
+            <h1>References</h1>
+            <p style={{
+                marginTop: "1rem",
+                marginBottom: "1rem",
+                fontStyle: "italic"
+            }}>
+                Click on a reference to add it to the document.
+            </p>
+
+            <ul style={{ borderWidth: "1px" }}>
+                {REFS_DATABASE.map((ref) => (
```

Comment:
Suggest spelling out `reference` since `ref` means something very specific in React.